build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+	id 'org.springframework.boot' version '2.1.3.RELEASE'
+	id 'java'
+}
+
+apply plugin: 'io.spring.dependency-management'
```

Comment:
I think we might need another plugin here to get the `./gradlew test` etc. commands working. Not 100% sure though.